compute/admin_guide/configure/agentless-permission-templates.adoc

```diff
@@ -0,0 +1,68 @@
+All scanning happens within the same account.
+This account is independent of other accounts.
+The account requires all  needed permissions to create snapshots, create scanners, etc. 
+The permission template you need to download ends in `_target_user_permissions.json`.
+
+==== Scan Hub Account
+
+This account is the hub account where you want all account scanning to happen.
+It scans not just the hosts in this account, but it also scans the hosts in other cloud accounts.
+This account requires the permissions needed to run Prisma Cloud Scanners and to access to all other cloud accounts.
+These permissions allow the hub account to see and transfer host snapshots from other accounts to this account. 
+The permission template you need to download ends in `_hub_user_permissions.json`.
+
+==== Target Account
+
+This is the account that a hub account scans.
+The permissions needed in target accounts are less than those required for independent scanning, but agentless still needs specific permissions to enable the hub account to scan a target account.
+The permission template you need to download ends in `_hub_target_user_permissions.json`.
+
+
+=== Azure
+
+Azure only requires individual account scanning permissions because there is no hub account. Prisma Cloud creates a dedicated PCC resource group, which enables easy cost calculation in the same account scanning. If you don't provide a security group, Prisma Cloud automatically creates one.
+
+=== GCP
+
+You need to download four permission templates as JINJA files for GCP cloud account scanning. 
+Three templates are needed to enable the different ways accounts are scanned: independent scanning, scan hub account, and target account.
+A fourth template is needed to enable the hub account to access the target account.
+
+
+==== Independent Scanning
+
+All scanning happens within the same account.
+This account is independent of other accounts.
+The account requires all  needed permissions to create snapshots, create scanners, etc. 
+The permission template you need to download ends in `_target_user_permissions.yaml.jinja`.
+
+==== Scan Hub Account
+
+This account is the hub account where you want all account scanning to happen.
+It scans not just the hosts in this account, but it also scans the hosts in other cloud accounts.
+This account requires the permissions needed to run Prisma Cloud Scanners and to access to all other cloud accounts.
+These permissions allow the hub account to see and transfer host snapshots from other accounts to this account. 
+The permission template you need to download ends in `_hub_user_permissions.yaml.jinja`.
+
+==== Target Account
+
+This is the account that a hub account scans.
+The permissions needed in target accounts are less than those required for independent scanning, but agentless still needs specific permissions to enable the hub account to scan a target account.
+The permission template you need to download ends in `_hub_target_user_permissions.yaml.jinja`.
```

Comment:
"Apply the permission template that ends with: `_hub_target_user_permissions.yaml.jinja` to the accounts you selected to be scanned by the Hub account.